lib/src/Base/Stat/openturns/CovarianceMatrix.hxx

```diff
@@ -67,8 +67,13 @@ public:
   using SymmetricMatrix::operator +;
 
   /** CovarianceMatrix multiplication (must have consistent dimensions) */
+#ifdef _MSC_VER // VS2010 does not like 'using' being called after overloads
+  using SymmetricMatrix::operator *;
+#endif
   CovarianceMatrix operator * (const IdentityMatrix & m) const;
+#ifndef _MSC_VER // VS2010 does not like 'using' being called after overloads
   using SymmetricMatrix::operator *;
+#endif
```

Comment:
You can switch the 2 lines unconditionnally:
```
  using SymmetricMatrix::operator *; 
  CovarianceMatrix operator * (const IdentityMatrix & m) const; 
```

BTW it is strange that there is no problem with `operator+` just above.